README.md

```diff
@@ -129,6 +129,61 @@ url | proxy-server.subman.example.com:3128
 username | proxyuser
 password | password
 
+Pipenv
+------
+
+There is experimental support for installation of subscription-manager using
+Pipenv. We support installing subscription-manager using Pipenv only on Fedora 30
+and later and RHEL/CentOS 8 and later. Python 3 is also required. It is necessary
+to install following packages to your system, because binary module have to be
+compiled in virtual environment:
+
+```bash
+dnf install -y pipenv gcc make python3-dev python3-devel \
+    openssl-devel intltool libnl3-devel
+```
+
+You can create virtual environment using following steps:
+
+1. Create virtual environment using Python 3 and it is necessary to
+   use `--site-packages` argument, because virtual environment has to
+   use `rpm` Python package installed in your system. It is not possible
+   to install `rpm` Python package to virtual environment using pip/pipenv.
+
+   ```bash
+   pipenv --site-packages --three
```

Comment:
have you tried with python 2 ? Does it work for similar steps?

Reply:
Interesting. I tried it now on CentOS 7 with Python 2.7 and it works. I had to change only first step. I will update README.md